unhandled exception

When I run pronterface/ printrun, error occures:

I have no clue on how to solve this but this is what console says:

Traceback (most recent call last):
  File "pronterface.py", line 62, in <module>
  File "printrun\pronterface.py", line 2581, in __init__
wx._core.wxAssertionError: C++ assertion ""lang != wxLANGUAGE_UNKNOWN"" failed at ..\..\src\common\intl.cpp(425) in wxLocale::Init(): Initializing unknown locale doesn't make sense, did you mean to use wxLANGUAGE_DEFAULT perhaps?

![pronter](https://user-images.githubusercontent.com/131581490/233827667-aa338a72-d607-4efc-afb7-157437a48159.jpg)


Can you help me with this? /- my email is > sujdovic21@gmail.com
Thanks.

--- a/printrun/pronterface.py
+++ b/printrun/pronterface.py
@@ -2578,6 +2578,10 @@ class PronterApp(wx.App):
     def __init__(self, *args, **kwargs):
         super(PronterApp, self).__init__(*args, **kwargs)
         self.SetAppName("Pronterface")
-        self.locale = wx.Locale(wx.Locale.GetSystemLanguage())
+        lang = wx.Locale.GetSystemLanguage()
+        # Fall back to English if unable to determine language
+        if lang == wx.LANGUAGE_UNKNOWN:
+            lang = wx.LANGUAGE_ENGLISH_US
+        self.locale = wx.Locale(lang)
         self.mainwindow = PronterWindow(self)
         self.mainwindow.Show()

